README.md

```diff
@@ -71,7 +71,7 @@ file and its purpose, all the modules have the same files, as described below:
 
  - **container.yaml**: used by OSBS builds.
  - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
- - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to master branch.
+ - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to main branch.
```

Comment:
```suggestion
 - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies.
```

Reply:
let's remove it, I have an idea to get rid of master branch of external repositories.